bagelbids/tests/test_cli_pheno.py

```diff
@@ -85,6 +92,50 @@ def test_get_columns_that_are_about_concept(test_data):
     assert [] == get_columns_about(data_dict, concept="does not exist concept")
 
 
+def test_map_columns(test_data):
+    """Test that inverse mapping of concepts to columns is correctly created"""
+    with open(test_data / "example2.json", "r") as f:
+        data_dict = json.load(f)
+
+    result = map_categories_to_columns(data_dict)
+
+    assert {"participant", "session", "sex"}.issubset(result.keys())
+    assert ["participant_id"] == result["participant"]
+    assert ["session_id"] == result["session"]
+    assert ["sex"] == result["sex"]
+
+
+def test_get_transformed_categorical_value(test_data):
+    """Test that the correct transformed value is returned for a categorical variable"""
+    with open(test_data / "example2.json", "r") as f:
+        data_dict = json.load(f)
+
+    assert "bids:Male" == get_transformed_values(
+        columns=["sex"],
+        row=pd.Series({"sex": "M"}, index=["sex"]),
```

Comment:
Why spell this out rather than loading in a row from example2.tsv?

Reply:
Yeah, good question. I was trying to find a balance between rewriting a lengthy example (like the data dictionary) and defining the data directly in the test for readability. My concern with doing what you suggest is that now I have to go and look at both examples to understand the test. But you already have to do this for the data dictionary, so it's inconsistent already. 

I'm OK with changing it to what you suggest (i.e. loading example.tsv directly and slicing in the test)

Reply:
Yeah, I think given the existence of the example tsv/csv pairings, it makes sense to use them together. To me, rewriting part of an example here implies that we're testing something different than is in the corresponding tsv/csv.